debug.log

```diff
@@ -0,0 +1 @@
+[1013/194149.139:ERROR:directory_reader_win.cc(43)] FindFirstFile: The system cannot find the path specified. (0x3)
```

Comment:
what is this o.o

Reply:
I have no idea what that is. I don't even know what debug.log is.